core/src/main/java/com/google/googlejavaformat/java/ModifierOrderer.java

```diff
@@ -152,7 +152,15 @@ private static void addTrivia(StringBuilder replacement, ImmutableList<? extends
    * is not a modifier.
    */
   private static Modifier asModifier(Token token) {
-    return getModifier(((JavaInput.Tok) token.getTok()).kind());
```

Comment:
WDYT about putting this in the `default:` in `getModifier`? This is currently the only call of `getModifier`, but putting the logic there would prevent a bug if another call of `getModifier` is added.

Also I'd consider using a switch instead of the if/else chain:

```java
switch (token.getTok().getText()) {
  case "sealed":
     return Modifier.valueOf("SEALED");
```

Reply:
I didn’t do that, as `getModifier` takes a `TokenKind` (and returns early if that’s `null`), but I agree that it’s not optimal.  Would it be OK if I inline `getModifier` in `asModifier` and handle both cases using switches there?  (I agree that a switch is much better than an if/else chain.)

Reply:
Inlining `getModifier` SGTM